composer.json

```diff
@@ -7,9 +7,10 @@
             "email": "michael@wearejh.com"
         }
     ],
+    "minimum-stability": "beta",
```

Comment:
Had to switch to beta to get console 4.0 which isn't yet released but has a pretty important fix regarding options and default values.